indigo_app/urls.py

```diff
@@ -87,6 +87,8 @@
     path('documents/<int:doc_id>/popup', documents.DocumentPopupView.as_view(), name='document_popup'),
 
     path('tasks/', tasks.UserTasksView.as_view(), name='my_tasks'),
+    path('tasks/project/', tasks.TaxonomyProjectListView.as_view(), name='taxonomy_project_list'),
```

Comment:
TaxonomyTopicTaskListView (on TaxonomyTaskListView) -- include the word Tasks to differentiate from other taxonomy list views.

Reply:
changed :+1: